src/ptf/packet.py

```diff
@@ -1,146 +1,20 @@
-# Distributed under the OpenFlow Software License (see LICENSE)
-# Copyright (c) 2010 The Board of Trustees of The Leland Stanford Junior University
-# Copyright (c) 2012, 2013 Big Switch Networks, Inc.
-"""
-Wrap scapy to satisfy pylint
+""" A pluggable packet module
```

Comment:
Will this change be backward compatible?

Reply:
Can you explain more in details what do you mean?
Scapy will remain as the default packet manipulation tool, the old unchanged `packet.py` will be kept as `packet_scapy.py`. Therefore, if you don't do anything, PTF will use Scapy.

Reply:
I tested this with a docker container(python:3), and it didn't work

To install PTF library with your change:
```bash
pip install git+https://github.com/sborkows/ptf
```

Got error when trying to load some basic class from ptf.packet:

```python
>>> import ptf
>>> from ptf.packet import Ether
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/ptf/packet.py", line 10, in <module>
    __module = __import__(config.get("packet_manipulation_module"), fromlist=["*"])
TypeError: __import__() argument 1 must be str, not None
```

Reply:
By backward compatible I mean I should be able to use `from ptf.packet import Ether` instead of using `from ptf.packet_scapy import Ether`

Reply:
I found this issue earlier, https://github.com/p4lang/ptf/pull/147#discussion_r699624503
There's a missing default value like a `packet_scapy.py`. Since Scapy is the most popular framework for this job and we need to keep backward compatible, Scapy is a must-have as the default packet generator.

Ping @sborkows

Reply:
Has this been resolved?

Reply:
Yes, I provided the default module as "ptf.packet_scapy"